src/orquestra/sdk/_base/_workflow.py

```diff
@@ -113,7 +115,14 @@ def model(self) -> ir.WorkflowDef:
         else:
             _futures = futures
 
-        return _traversal.flatten_graph(self, _futures)
+        model = _traversal.flatten_graph(self, _futures)
+
+        if len(model.task_invocations) < 1:
+            raise WorkflowSyntaxError(
+                f"The workflow '{model.name}' requires 0 tasks to be executed. "
```

Comment:
Suggestion: TBH I was a bit confused at first by this text. Maybe something more straight user-facing? Like "Your wofklow definition (or the workflow you are trying to submit? Whatever works better for you) [...]." 
But this thing here works for me as well.

Reply:
Good thought, thanks! [b383015](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/86/commits/b383015b8e90c27bae6045f6859dfd86de99141e) updates this to be a little more user-facing and also include the file and line number where the definition is found to help point the user to the changes they need to make. It now reads:

```
"The workflow '[wf name]' (defined at [file] line [line]) cannot be submitted as it does not define any tasks to be executed. Please modify the workflow definition to define at least one task and retry submitting the workflow."
```